sachungo/movies
# Retrieve a list of movies

## Issue
https://www.pivotaltracker.com/n/projects/2189922

## What has been done
- Requested a list of movies from https://api.themoviedb.org/3/
- Displayed the movies on a page

## How can it be tested
- Navigate to the heroku PR link in the comments below to view the page in the browser, you should be able to see a list of movies on the page

## Screenshot
![image 2018-08-20 at 11 21 41 am](https://user-images.githubusercontent.com/12892171/44329211-87688f80-a46c-11e8-8054-9cdef8a98d3d.png)
